secure-biometric-interface-integrator/stories/Sbd.stories.js

```diff
@@ -0,0 +1,199 @@
+      table: {
+        // category: "Design",
+        defaultValue: {
+          summary: JSON.stringify(customStyle),
+        },
+      },
+    },
+    sbiEnv: {
+      control: "object",
+      description: "For customization of SBI environment.",
+      table: {
+        // category: "Settings",
+        defaultValue: { summary: JSON.stringify(sbiEnv) },
+      },
+    },
+    onCapture: {
+      control: "events",
+      type: { required: true },
+      description:
+        "The function to be called with Biometric response (successful/failed capture response)",
+      // table: { category: "Required" },
+    },
+    onErrored: {
+      control: "events",
+      description:
+        "Optional callback function on capture failure with the error code string (likely due to timeout)",
+      // table: { category: "Error" },
+    },
+  },
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+
+const renderComponent = (args) => {
+  const container = document.createElement("div");
+  container.setAttribute("id", "secure-biometric-interface-integration");
+  args.container = container;
+  return div(
+    {
+      style: {
+        width: "100%",
+        background: "#f6f6f2",
+      },
+    },
+    div(
+      {
+        style: {
+          margin: "auto",
+          width: "50%",
+          border: "3px solid #d8d8d8",
+          padding: "50px",
+          display: "flex",
+          "align-items": "center",
+          "justify-content": "center",
+        },
+      },
+      init(args)
+    )
+  );
+};
+
+export const Primary = {
+  args: {
+    buttonLabel: "scan_and_verify",
+    disable: false,
+    langCode: "en",
+    sbiEnv,
+    customStyle,
+    onCapture: (e) => action("onCapture")(e),
+    onErrored: (e) => action("onErrored")(e),
+  },
+};
+
+Primary.parameters = {
+  docs: {
+    source: {
+      code: `
+      <div id="secure-biometric-interface-integration"></div>
+
+      // in javascript
+      SecureBiometricInterface.init({
+        container: document.getElementById("secure-biometric-interface-integration"),
+        buttonLabel: "scan_and_verify",
+        sbiEnv: {
+          env: "Staging",
+          captureTimeout: 30,
+          irisBioSubtypes: "UNKNOWN",
+          fingerBioSubtypes: "UNKNOWN",
+          faceCaptureCount: 1,
+          faceCaptureScore: 70,
+          fingerCaptureCount: 1,
+          fingerCaptureScore: 70,
+          irisCaptureCount: 1,
+          irisCaptureScore: 70,
+          portRange: "4501-4502",
+          discTimeout: 6,
+          dinfoTimeout: 30,
+          domainUri: \`${window.origin}\`,
+        },
+        langCode: "en",
+        disable: false,
+        transactionId: "123456789",
+        onCapture: (e) => {
+          console.log("*******************gettiing the biometric response");
+          console.log(e);
+        },
+        onErrored: (e) => {
+          console.log("**********getting error from secure bio device");
```

Comment:
"Error OnCapture - SBI returned error" Also should this be console.log or console.error or console.warn ?

Reply:
hi @gsasikumar , change the text of error & used console.debug() for the response, check this [044194a](https://github.com/mosip/esignet-plugins/pull/5/commits/044194a5882341fd950e519ef9b08f59144e6caa) commit